src/battle_ai_main.c

```diff
@@ -3324,7 +3333,10 @@ static u32 AI_CalcMoveScore(u32 battlerAtk, u32 battlerDef, u32 move)
         if (gBattleMons[battlerDef].statStages[STAT_EVASION] < 7 || aiData->abilities[battlerAtk] == ABILITY_NO_GUARD)
             ADJUST_SCORE(-2);
         break;
-	case EFFECT_BIDE:
+    case EFFECT_SPICY_EXTRACT:
+        // TODO: Make IncreaseStatDownScore function, just like IncreaseStatUpScore
```

Comment:
Recommend using if the attacker has Foul Play. If the attacker doesn't have Foul Play, then it shouldn't be used if the enemy is a physical attacker (you don't want to buff the target in a bad way for you).

Reply:
That one I forgot but I'll make sure to follow up on it.

Reply:
added